doc/explanation/networks.md

```diff
@@ -1,7 +1,107 @@
+# About networking
+
+There are different ways to connect your instances to the Internet. The easiest method is to have LXD create a network bridge during initialization and use this bridge for all instances, but LXD supports many different and advanced setups for networking.
+
+## Network devices
+
+Every instance must be assigned at least one network device, also called {abbr}`NIC (Network Interface Controller)`.
+You can configure the network device in one of the following ways:
+
+- Use the default network bridge that you set up during the LXD initialization.
+  Check the default profile to see the default configuration:
+
+        lxc profile show default
+
+  This method is used if you do not specify a network device for your instance.
+- Use an existing network interface by adding it as a network device to your instance.
+  This network interface is outside of LXD control.
+  Therefore, you must specify all information that LXD needs to use the network interface.
+
+  Use a command similar to the following:
+
+        lxc config device add <instance_name> <device_name> nic nictype=<nic_type> ...
+
+  See {ref}`instance_device_type_nic` for a list of available NIC types and their configuration properties.
+
+  For example, you could add a pre-existing Linux bridge (`br0`) with the following command:
+
+        lxc config device add <instance_name> eth0 nic nictype=bridged parent=br0
+- {doc}`Create a managed network </howto/network_create>` and add it as a network device to your instance.
+  With this method, LXD has all required information about the configured network, and you only need to specify the network name when adding the device:
+
+        lxc config device add <instance_name> <device_name> nic network=<network_name>
+
+  If needed, you can add further properties to the command to override the default settings for the network.
+
+## Managed networks
+
+Managed networks in LXD are created and configured with the `lxc network` command.
+
+Depending on the network type, LXD either fully controls the network or just manages an external network interface.
+
+Note that not all {ref}`NIC types <instance_device_type_nic>` are supported as network types.
+LXD can only set up some of the types as managed networks.
+
+### Fully controlled networks
+
+Fully controlled networks provide most functionality, including, for example, the ability to do IP management.
```

Comment:
In the external networks section below it says:

> External networks use network interfaces that already exist.

Perhaps here then it should indicate the opposite, that:

> Fully controlled networks create new network interfaces and provide most functionality, including, for example, the ability to do IP management.